src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -266,5 +267,58 @@ public override int GetHashCode()
                 Hash.Combine(base.GetHashCodeHelper(),
                 Hash.Combine((int)this.SpecifiedLanguageVersion, 0));
         }
+
+        /// <summary>
+        /// Parse a LanguageVersion from a string input, as the command-line compiler does.
+        /// </summary>
+        public static bool TryParseLanguageVersion(string version, out LanguageVersion result)
```

Comment:
`LanguageVersion` is an enum. 
See discussion and options above (https://github.com/dotnet/roslyn/pull/17435#discussion_r104018309)